src/game/main.cpp

```diff
@@ -162,6 +173,201 @@ void Check_Windowed(int argc, char *argv[])
 #endif
 }
 
+#if defined PLATFORM_WINDOWS && defined GAME_DLL
+// zh: 0x00401000
+static LRESULT __stdcall Wnd_Proc(HWND window_handle, UINT message, WPARAM w_param, LPARAM l_param)
+{
+    if (g_theIMEManager != nullptr && g_theIMEManager->Service_IME_Message(window_handle, message, w_param, l_param)) {
+        return g_theIMEManager->Result();
+    }
+
+    CopyProtect::checkForMessage(message, l_param);
```

Comment:
Does CopyProtect need calling in Thyme? Does it do anything for us?

Reply:
Its just an empty function so it does nothing. I think it makes sense to just leave it calling it like this for posterity.